docs/learn.rst

```diff
@@ -1,22 +1,26 @@
-Learner
-#######
+.. _learn:
+
+Learn with Kolibri
+##################
+
+.. _access_learner:
 
 Access Kolibri
 ==============
 
 .. note::
   	* If you are using Kolibri in your school, education center or facility, your coach or administrator will provide the instructions how to open the sign-in page, and username and password if necessary.
-  	* If you are using Kolibri on your own, outside an education center or facility, follow the instructions how to :doc:`access` according to your operating system. 
+  	* If you are using Kolibri on your own, outside an education center or facility, follow the instructions how to :doc:`access` according to your operating system.
 
-To sign in to **Kolibri** and start learning follow these steps:
+To sign in **Kolibri** and start learning follow these steps:
```

Comment:
There was a whole topic about that on SE English, and I'm pretty sure `to` was not required for this phrase to be grammatically correct. I'll try to find the link...

Reply:
I will defer to SE english :)

Reply:
@indirectlylit not gonna spend down tracking the source of this, but there were some conflicts where you had corrected language in the same lines as @radinamatic and I got blurry eyed trying to see how to merge them, this could easily be one of them. Sorry in advance.

I will wrap up this whole PR as best as possible in a single commit tomorrow, thanks for all the good comments!